app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Home | Ruby on Rails Tutorial Sample
+App</title>
+</head>
+<body>
+<h1>Sample App</h1>
+<p>
+This is the home page for the
+<a href="http://www.railstutorial.org/">Ruby on
+Rails Tutorial</a>
+sample application.
```

Comment:
1. Tìm hiểu và sử dụng I18n cho toàn bộ text trên view
Self check các file html

2. Dùng là link_to thấy vì thẻ a của html

Reply:
.